style.css

```
@@ -0,0 +1,358 @@
+@keyframes appearMenu {
+  0% {
+    transform: scale(1.2);
+    opacity: 0;
+  }
+  100% {
+    transform: scale(1);
+    opacity: 1;
+  }
+}
+
+@keyframes appear {
+  0% {
+    opacity: 0;
+    color: green;
+    transform: translate(-50px, 0);
+  }
+  100% {
+    opacity: 1;
+    color: white;
+    transform: translate(0, 0);
+  }
+}
+
+@keyframes disappearUl {
+  0% {
+    opacity: 1;
+  }
+  50% {
+    transform: translate(-10px, 0px);
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes disappear {
+  0% {
+    opacity: 1;
+    transform: translate(0px, 0);
+  }
+  100% {
+    color: red;
+    opacity: 0;
+    transform: translate(-50px, 0);
+  }
+}
+
+@keyframes curtainDown {
+  0% {
+    opacity: 0;
+    transform: translate(0px, -50px);
+  }
+  100% {
+    opacity: 1;
+    transform: translate(0px, 0);
+  }
+}
+
+@keyframes curtainUp {
+  0% {
+    opacity: 1;
+    transform: translate(0px, 0px);
+  }
+  100% {
+    opacity: 0;
+    transform: translate(0px, -50px);
+  }
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background-image: url("https://blog.justinablakeney.com/wp-content/uploads/2017/08/Aja-2-holeweb.jpg");
+  color: white;
+  background-attachment: fixed;
+}
+
+h1 {
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 40px;
+}
+
+#list-disappear {
+  animation: disappearUl 0.5s ease 0s 1 normal;
+}
+
+#subButton {
+  transition: 0.5s;
+}
+
+#subButton:hover {
+  transform: scale(1.1);
+}
+
+#deleteBtn {
+  transition: 0.5s;
+}
+
+#deleteBtn:hover {
+  transform: scale(1.1);
+}
+
+main {
+  width: 60%;
+  height: 90%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 100px 20px 100px 20px;
+  border: rgb(142, 200, 178) 15px solid;
+  background-color: rgb(29, 83, 119);
+  box-shadow: 0px 1px 10px black;
+  overflow: hidden;
+}
+
+main form {
+  display: flex;
+  padding: 10px;
+  width: 100%;
+  justify-content: center;
+}
+
+#saveBtn {
+  height: 30px;
+  border: 2px solid;
+  background: white;
+  border-radius: 5px;
+}
+
+#seeMoreBtn:hover {
+  transition: 0.5s;
+  transform: scale(1.1);
+}
+
+#seeMoreDiv {
+  margin-top: 0px;
```

Isto está bem, é só uma nota, valores 0, podes simplificar com:
```suggestion
  margin-top: 0;
```

O mesmo para todas as outras propriedades:
```css
box-shadow: 1px 0 100px red;
```